### PR TITLE
[Docs] Refactor docs

### PR DESCRIPTION
**Pull request by zhouzaida**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Refactor the documentation. Documentation can be visited at https://mmcv-refactor-docs.readthedocs.io/en/latest/

## Modification
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by zhouzaida** [comment]
Documentation can be visited at https://mmcv-refactor-docs.readthedocs.io/en/latest/

File: docs/understand-the-mmcv/registry.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ converter_cfg = dict(type='Converter1', a=a_value, b=b_value)
 converter = CONVERTERS.build(converter_cfg)
 ```
 
-## Customize Build Function
```
Comment (ZwwWayne) [inline docs/understand-the-mmcv/registry.md:65]: understand-the-mmcv -> understand_mmcv or simply use tutorials would be fine.

File: docs/community/contributing.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+## Contributing to OpenMMLab
+
```
Comment (ZwwWayne) [inline docs/community/contributing.md:2]: Downstream OpenMMLab projects all rely on the CONTRIBUTING.md at https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md

File: docs/community.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Community
```
Comment (ZwwWayne) [inline docs/community.rst:1]: Community -> Notes.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
-# Contributing to OpenMMLab
+## Contributing to OpenMMLab
```
Comment (innerlee) [inline CONTRIBUTING.md:1]: Why?

Comment (zhouzaida) [reply CONTRIBUTING.md:1]: Changing # to ## makes us directly create a symbolic link to docs  because those articles in docs all start with ##

Comment (innerlee) [reply CONTRIBUTING.md:1]: Random question: what if changing ## to # for files in docs?

Comment (zhouzaida) [reply CONTRIBUTING.md:1]: it will break the structure of docs

File: docs/index.rst [hunk 5]
```diff
@@ -1,26 +1,15 @@
...
-   cnn.md
-   ops.md
-   build.md
+   getting-started.rst
    deployment.rst
-   trouble_shooting.md
+   understand-mmcv.rst
    api.rst
-
+   trouble_shooting.md
```
Comment (ZwwWayne) [inline docs/index.rst:11]: trouble_shooting -> FAQ, also change the title from trouble shooting to Frequently Asked Questions.

File: docs/getting-started.rst [hunk 6]
```diff
@@ -0,0 +1,9 @@
+Getting started
```
Comment (ZwwWayne) [inline docs/getting-started.rst:1]: getting-started -> getting_started.rst

Comment (zhouzaida) [reply docs/getting-started.rst]: done

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**